sdk-cpp/include/do_download_property.h

```diff
@@ -45,56 +49,48 @@ enum class download_property
     integrity_check_info,           // std::string
     integrity_check_mandatory,      // boolean
     total_size_bytes,               // uint64
-    // Fe and beyond
+
+    // For the COM interface, the following properties are available only in Windows 21H1 and beyond
```

Comment:
>21H1 [](http://example.com/codeflow?start=85&length=4)

21H1 is still the Vibranium build. Let's go with the build number for Cobalt here (22000) since that is the first desktop release with support for these properties.

Reply:
modified comment